src/cloud/iotile-cloud-serv.ts

```diff
@@ -629,6 +651,24 @@ export class IOTileCloud {
     });
   }
 
+
+  public async fetchNotes(targetSlug: string): Promise<Array<Note>> {
```

Comment:
As described above, please add the apiFilter so we can use this function with other filters (i.e. date ranges)